<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mask Editor</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox_Close">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>590</y>
     <width>120</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ResetMask">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>200</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>RESET Mask</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_GenerateMask">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>99</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>GENERATE Mask</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_MaskAutoSave">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>156</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Auto-save</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>191</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>Drawing</string>
   </property>
   <widget class="QGroupBox" name="groupBox_DrawingColor">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>28</y>
      <width>171</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Line Color</string>
    </property>
    <widget class="QRadioButton" name="radioButton_DrawingColor_Red">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>95</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Red</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_DrawingColor_Orange">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>53</y>
       <width>95</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Orange</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_DrawingColor_Yellow">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>81</y>
       <width>95</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Yellow</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_DrawingColor_Green">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>25</y>
       <width>95</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Green</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_DrawingColor_Blue">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>53</y>
       <width>95</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Blue</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_DrawingColor_Purple">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>81</y>
       <width>95</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Purple</string>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="checkBox_FillPolygon">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>161</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Fill Polygon</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_AddMask">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>37</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>ADD Mask</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>285</y>
     <width>410</width>
     <height>280</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Masks</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox_Close</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox_Close</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
